recipes/default.rb

```diff
@@ -17,3 +17,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+case node['platform_family']
```

Comment:
another alternative is to use something similar to:

unless node['platform'] == 'ubuntu'
  Chef::Log.info("Skipping as running on unsupported platform #{node['platform']}")
  return
end

which ive seen in many recipes from the community. I just want to make sure this, even if included in a role used on hybrid environments (mixed windows and other OSes) it only gets executed on windows and not other OSes as it gives errors. i have the issue with one role that has support for both windows and ubuntu flavors and if i include this recipe in the role it error on linux and i find it a waste of code and time maintaining a wrapper cookbook for a few lines of code.

i am open to ideas however.